app/controllers/katello/api/registry/registry_proxies_controller.rb

```diff
@@ -293,6 +301,11 @@ def v1_ping
     end
 
     def v1_search
+      if request.headers['HTTP_USER_AGENT'].downcase.include?('libpod')
```

Comment:
Everything looks good and seems to work well.  Could we add a comment here explaining that podman checks explicitly for a 404 before trying to call /catalog  ?

Reply:
You bet!